### PR TITLE
Minor cleanup

### PR DESCRIPTION
**Pull request by stevenschlansker**

### CODE REVIEW

**Comment by pennello** [comment]
Taking a look now.

File: src/test/java/com/opentable/kafka/EmbeddedZookeeperTest.java [hunk 1]
```diff
@@ -52,7 +52,8 @@ public void testBoots() throws Exception {
         final String connect = zk.getConnectString();
         ZooKeeper client = new ZooKeeper(connect, 60000, this);
 
-        while (!client.getState().isConnected()) {
+        final long since = System.currentTimeMillis();
+        while (events.isEmpty() && System.currentTimeMillis() - since < 30_000) {
```
Comment (pennello) [inline src/test/java/com/opentable/kafka/EmbeddedZookeeperTest.java]: I know this is pretty minimal test code, but I've been enjoying a countdown latch, which handles the spurious wakeups for you.

File: src/main/java/com/opentable/kafka/embedded/EmbeddedKafkaBroker.java [hunk 2]
```diff
@@ -336,4 +335,8 @@ private Properties baseProducerProperties() {
         props.put("bootstrap.servers", getKafkaBrokerConnect());
         return props;
     }
+
+    interface RetryAction {
+        void run() throws ExecutionException, InterruptedException, KafkaException, TimeoutException;
+    }
```
Comment (pennello) [inline src/main/java/com/opentable/kafka/embedded/EmbeddedKafkaBroker.java]: Oh, very good.  We should have complained the first time we saw as much duplicated!

File: src/test/java/com/opentable/kafka/EmbeddedZookeeperTest.java (hunk 1 above)
Comment (stevenschlansker) [reply src/test/java/com/opentable/kafka/EmbeddedZookeeperTest.java]: Yeah, I agree.  But I'm going to keep the scope limited right now.  We can replace that later for sure, especially if we add more tests that use this.

Comment (pennello) [reply src/test/java/com/opentable/kafka/EmbeddedZookeeperTest.java]: Yup.  Sounds good.

**Pull request merged**